File: Flow Control/function6.js
function main() {
    for(let i = 0; i < 10; i++) {
        count('#', i*2)
    }
}

function count(ch, numero) {
    let print = ''
    for(let x = 0; x < numero; x++){
        print += ch
    }
    console.log(print)
}


main()

File: Flow Control/function6.c
#include <stdio.h>
#define MAX 11
void bar(char c, int count);

int main()
{
    for (int i = 1; i < MAX; i++)
    {
        bar('*', i * 2);
    }
        return(0);
}

void bar(char c, int count)
{
    for ( int i = 0; i < count; i++){
        putchar(c);
    }
    putchar('\n');
    
}